### ISSUE TITLE
[GUI][Visual bug] Shortened topbar icon shows full sync although wallet is still syncing

### ISSUE THREAD

**Issue by NoobieDev12** [PIVX-Project/PIVX#1421]
As title says and as visible on screenshots below, shortened icon is showing like wallet is fully synced, although syncing is still in progress. I can't say exactly, but it seems like shortened icon is filling from 0 to 100% while counting blocks from 0 to 400k on mainnet (shortened icon loadbar "thinks" full sync is somewhere at block ~400k on mainnet and ~100k on testnet).

![syncProgressBug1](https://user-images.githubusercontent.com/57232689/76810131-4faefe80-67ed-11ea-8bc9-b3b462559da8.jpg)

![syncProgressBug2](https://user-images.githubusercontent.com/57232689/76810134-52115880-67ed-11ea-9c3e-4b36f9fd4d45.jpg)

**Comment by akshaynexus** [comment]
this looks like a ratio/progress calc issue,is the issue still happening on latest master branch builds?

**Comment by NoobieDev12** [comment]
@akshaynexus yes, still happening. You can try yourself on testnet by removing all block files and start syncing from scratch. Around block 100k shortened progress bar is already full.
